src/reference.yaml

```diff
@@ -1401,9 +1401,9 @@ paths:
           description: City.
         - name: postal_code
           in: formData
-          required: false
+          required: true
```

Comment:
@maria-thom If this parameter if optional in some cases, rather than being always mandatory, required should not be "true"

Reply:
@jonathancouchman I did consider this.  We're trying to make it clearer for Outsourced-KYC clients which fields are mandatory for them.  It's a difficult decision whether it's better to list them as required and adding a comment that it's only for some accounts, or making it optional and adding a comment that it's required for some accounts.  I decided that it was clearer to list them as required with a comment as users may not pay too much attention to the optional section.  Is what you're suggesting an agreed standard?

Reply:
@maria-thom the swagger 2.0 specification is here https://swagger.io/docs/specification/2-0/describing-parameters/

_if a value is required, the client must always send it_